docs/flutter-doxygen.css

```diff
@@ -0,0 +1,7 @@
+div.contents {
+    margin-bottom: 10px;
+    padding: 12px;
+    margin-left: 12px;
+    margin-right: auto;
+    max-width: 960px;
+}
```

Comment:
Newline at end of file please.

Reply:
Done, Visual Studio Code removes ending newlines in css files.  I'm guessing for filesize.